pkg/controllers/data.go

```diff
@@ -49,64 +49,79 @@ func (d *DataController) writeResponse(w http.ResponseWriter, statusCode int, ms
 
 // List handles incoming data list requests
 func (d *DataController) List(w http.ResponseWriter, r *http.Request) {
+	d.logger.Debug("get data request received")
```

Comment:
Do you know if request parameters are already being logged (by the `mux`, for example)? I think for debug level it'd be useful.

Reply:
Actually, mux logs the requested endpoint, including the query parameters. But, either way, I added a log for the query (on debug mode).